CHANGELOG.rst

```diff
@@ -11,6 +11,28 @@
 Changelog
 +++++++++
 
+0.14.0
+======
+
+- Obey the ``--skip-subprojects`` option passed to the ``meson
+  install`` command.
+- Implement support for building wheels targerting the Python limited
+  API. Extension modules targeting the Python limited API can be
+  easily build starting with the upcomin Meson 1.3.0 release.
+- Instead than silently using the version declared in ``meson.build``,
+  raise an error when ``pyproject.toml`` does not contain a
+  ``version`` field and ``version`` is not declared dynamic.
+- Fix the mtime of source files in the sdist tarball.
+- Fix the installation of files via ``install_subdir()``. For the
```

Comment:
I suggest extending the first sentence with _"when the `exclude_directories or `exclude_files keywords are used"_. The way it reads now, a user may think there was a broader problem.

Reply:
I thought that there were broader problems with handling `install_subdir()` but looking at the old code again, I cannot identify what these problems may have been. I'll update this entry.

Reply:
Nothing important I think - both Pandas and NumPy use it fairly extensively, and it didn't give them problems at least AFAIK.